src/status_im2/contexts/communities/discover/view.cljs

```diff
@@ -117,7 +116,7 @@
            featured-communities
            view-type
            selected-tab]}]
-  [react/animated-view
+  [rn/animated-view
```

Comment:
Unrelated, but is this animated view needed here? Seems to have no animated styles.